sections/sprint-1/pozyx.tex

```diff
@@ -0,0 +1,42 @@
+\section{Pozyx}
+Pozyx is a hardware/software solution that is used to provide positioning with an accuracy of down to 10 cm \cite{pozyx}.
+It makes use of ultra-wideband in combination for positioning, which according to their documentation is more precise and effecient than traditional positioning systems such as WiFi, bluetooth, RFID and GPS.
+
+Since the two major requirements for the positioning in this project are precision and a high update rate to ensure that the players are able to have reliable data available, the Pozyx system seems like a good place to start.
```

Comment:
```suggestion
Since the two major requirements for positioning in this project are precision and a high update rate in order to ensure that the players are able to have reliable data available, the Pozyx system seems like a good place to start.
```

Reply:
Seems like a good place to start er sådan lidt underligt formuleret synes jeg. Hvor går vi så hen hvis vi bare starter der??